components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -984,9 +984,28 @@ void RewardsServiceImpl::FetchWalletProperties() {
 
 void RewardsServiceImpl::FetchGrant(const std::string& lang,
     const std::string& payment_id) {
-  ledger_->FetchGrant(lang, payment_id);
+#if !defined(OS_ANDROID)
+  ledger_->FetchGrant(lang, payment_id, "");
+#else
+  safetynet_check::ClientAttestationCallback attest_callback = base::BindOnce(&RewardsServiceImpl::FetchGrantAttestationResult, base::Unretained(this),
+                                                                                lang, payment_id);
+  safetynet_check_runner_.performSafetynetCheck("", std::move(attest_callback));
+#endif
 }
 
+#if defined(OS_ANDROID)
+void RewardsServiceImpl::FetchGrantAttestationResult(const std::string& lang, const std::string& payment_id,
```

Comment:
this looks like a line length issue, can you check this against cpplint? Easiest way might be to run `npm run lint` from brave-browser

Reply:
Aligned code according to lint requirements